python/cdp_san_jose_backend/clean_database.py

```diff
@@ -0,0 +1,48 @@
+from cdp_backend.database import models as db_models
+import fireo
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.firestore import Client
+
+# Connect to the database
+fireo.connection(client=Client(
+    project="cdp-san-jose-5d9db455",
+    credentials=AnonymousCredentials()
+))
```

Comment:
Can't really test this but just trust me. Can you make a connection with credentials like so:

```python
fireo.connection(from_file="google-creds.json")
```

When this script will be ran, it will have access to those creds. In the correct directory.

Reply:
Actually, ignore this. I will change it right before we run it.